db/schema.rb

```diff
@@ -0,0 +1,57 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_11_24_154729) do
+  create_table "comments", force: :cascade do |t|
+    t.string "text"

```

Comment:
The `text` attribute of comments should have data type `text` instead of `string` because `text` has more characters to store and space than `string`.